x-pack/plugins/observability/public/pages/rules/index.tsx

```diff
@@ -193,17 +192,17 @@ export function RulesPage() {
         name: STATUS_COLUMN_TITLE,
         sortable: true,
         render: (_enabled: boolean, item: RuleTableItem) => {
-          return (
-            <StatusContext
-              disabled={!item.isEditable || !item.enabledInLicense}
-              item={item}
-              onStatusChanged={() => reload()}
-              enableRule={async () => await enableRule({ http, id: item.id })}
-              disableRule={async () => await disableRule({ http, id: item.id })}
-              muteRule={async () => await muteRule({ http, id: item.id })}
-              unMuteRule={async () => await unmuteRule({ http, id: item.id })}
-            />
-          );
+          return triggersActionsUi.getRuleStatusDropdown({
+            rule: item,
+            enableRule: async () => await enableRule({ http, id: item.id }),
+            disableRule: async () => await disableRule({ http, id: item.id }),
+            onRuleChanged: () => reload(),
+            isEditable: item.isEditable && isRuleTypeEditableInContext(item.ruleTypeId),
+            snoozeRule: async (snoozeEndTime: string | -1, interval: string | null) => {
```

Comment:
`interval` is not used. Should we keep it in the function signature?

Reply:
@fkanout Good catch. I used it, since it was used here https://github.com/elastic/kibana/pull/130721/files#diff-0b08d93b140ecb5d9343573bcb0c0fc3b18ca62347555259b247c9cf36b02e5cL627, but looks like this part of code was later refactored and interval indeed is not used. I will removed.

Reply:
@fkanout I removed unused `interval`